src/app/types/splTransfer.ts

```diff
@@ -10,8 +10,8 @@ export type SplTransfer = {
   sender: string;
   feeInLamports?: string;
   feeInSpl?: string;
-  unsignedTransactionBytes?: string;
-  signedTransactionBytes?: string;
+  unsignedTransactionBytes: Buffer;
+  signedTransactionBytes?: Buffer;
```

Comment:
what's the benefit of having this be a Buffer instead of a uint8array? or a base64 encoded string?

Reply:
At first, I thought uint8array. AI suggested Buffer, which seemed more generic. I'm okay either/or.

I take a stance on not being string. It's all bytes at the end of the day, but the indicated primitive inscribes its intended purpose. In this case, we hold serialized bytes instead of a message in prose.

Reply:
yeah i don't really have a strong preference either way (buffer v uint8array). Can't think of a specific reason that's really strong so I defer to our AI overloards as well